### PR TITLE
Bug - KB-5177

### PR DESCRIPTION
**Pull request by venkykandagaddala**

### CODE REVIEW

File: library/ws-widget/collection/src/lib/_common/discussions/discussions.component.ts [hunk 1]
```diff
@@ -31,7 +31,11 @@ export class DiscussionsComponent implements OnInit {
 
   handleSelectedDiscuss(discussData: any, context: string | boolean): void {
     const subType = context && context === 'my-discussions' ? 'my-discussions' : 'trending-discussions'
-    this.router.navigateByUrl(`/app/discussion-forum/topic/${discussData.slug}?page=home`)
+    let slug = discussData.slug
```
Comment (vishnubansaltarento) [inline library/ws-widget/collection/src/lib/_common/discussions/discussions.component.ts]: put in condition for discussData.slug

Comment (venkykandagaddala) [reply library/ws-widget/collection/src/lib/_common/discussions/discussions.component.ts]: updated

File: library/ws-widget/collection/src/lib/_common/discussions/discussions.component.ts [hunk 2]
```diff
@@ -31,7 +31,11 @@ export class DiscussionsComponent implements OnInit {
 
   handleSelectedDiscuss(discussData: any, context: string | boolean): void {
     const subType = context && context === 'my-discussions' ? 'my-discussions' : 'trending-discussions'
-    this.router.navigateByUrl(`/app/discussion-forum/topic/${discussData.slug}?page=home`)
+    let slug = (discussData && discussData.slug) ? discussData.slug : ""
+    if (!slug && discussData.topic) {
```
Comment (vishnubansaltarento) [inline library/ws-widget/collection/src/lib/_common/discussions/discussions.component.ts]: put a condition here as well 

!slug && discussData && discussData.topic

Comment (venkykandagaddala) [reply library/ws-widget/collection/src/lib/_common/discussions/discussions.component.ts]: update here too

**Pull request merged**